lib/src/js/value/color.dart

```diff
@@ -86,4 +447,30 @@ class _Channels {
   external double? get whiteness;
   external double? get blackness;
   external double? get alpha;
+  external double? get a;
+  external double? get b;
+  external double? get x;
+  external double? get y;
+  external double? get z;
+  external double? get chroma;
+}
+
+@JS()
+@anonymous
+class _ConstructionOptions extends _Channels {
+  external String? get space;
+}
+
+@JS()
+@anonymous
+class ChannelOptions {
```

Comment:
These new classes should be private, since they aren't intended to be used outside this library.